### PR TITLE
fix: prevent the newly created channel from being filtered out from ChannelList

### PR DESCRIPTION
**Pull request by AhyoungRyu**
Fixes https://sendbird.atlassian.net/browse/CLNP-3607

https://github.com/sendbird/sendbird-uikit-react/assets/10060731/3d19dc9e-a1f1-46b6-8196-09a9ecbdeec4

### CODE REVIEW

File: src/utils/index.ts [hunk 1]
```diff
@@ -947,3 +948,10 @@ export const arrayEqual = (array1: Array<unknown>, array2: Array<unknown>): bool
 export const isSendableMessage = (message?: BaseMessage | null): message is SendableMessageType => {
   return Boolean(message) && 'sender' in (message as SendableMessage);
 };
+
+/**
+ * If the channel is just created, the channel's createdAt and currentUser's invitedAt are the same.
+ */
+export const isChannelJustCreated = (channel: GroupChannel): boolean => {
+  return isSameSecond(channel.createdAt, channel.invitedAt);
```
[the remark this answers is not in the document]
Comment (chrisallo) [reply src/utils/index.ts:956]: 어떻게 하는게 좋을지 조금 생각을 해봤는데 `channel.messageOffsetTimeout`와 `channel.lastMessage?.createdAt`을 비교하는것도 한가지 방법일것같아요. lastMessage가 없거나 lastMessage.createdAt이 messageOffsetTimeout보다 작은 경우 막 생성된 채널로 보는것도 좋을것 같습니다.

File: src/modules/ChannelList/dux/__tests__/reducers.spec.js [hunk 2]
```diff
@@ -906,4 +919,51 @@ describe('Channels-Reducers', () => {
       expect(hideWithPreventAutoOnPreventAutoState.allChannels.length).toEqual(preventAutoUnhideParamsState.allChannels.length + 1);
     });
   });
+  describe.only('isChannelJustCreated', () => {
```
Comment (HoonBaek) [inline src/modules/ChannelList/dux/__tests__/reducers.spec.js:922]: need to remove `only`

File: src/modules/ChannelList/dux/__tests__/reducers.spec.js [hunk 3]
```diff
@@ -906,4 +919,51 @@ describe('Channels-Reducers', () => {
...
+    it('should add a newly created channel to the ChannelList if it is not just created', () => {
+      const newChannel = {
+        ...creatingChannel,
+        members: [user1, user2],
+        createdAt: 123456789,
+        invitedAt: 129999967,
+        lastMessage: null,
+      };
+      (isChannelJustCreated).mockReturnValue(false);
+      (getNextChannel).mockReturnValue(newChannel);
```
Comment (HoonBaek) [inline src/modules/ChannelList/dux/__tests__/reducers.spec.js:955]: doesn't need to wrap the variables with `()`?

**Review by HoonBaek — APPROVED** [review]
LGTM

File: src/modules/ChannelList/dux/__tests__/reducers.spec.js (hunk 3 above)
Comment (AhyoungRyu) [reply src/modules/ChannelList/dux/__tests__/reducers.spec.js]: Oh right. My bad

File: src/modules/ChannelList/dux/__tests__/reducers.spec.js (hunk 2 above)
Comment (AhyoungRyu) [reply src/modules/ChannelList/dux/__tests__/reducers.spec.js]: https://github.com/sendbird/sendbird-uikit-react/pull/1124/commits/d691222b5b7c6e7f4cd78767cceb1a13579c972f#diff-9536f4716bc6385ea2e3e35374ac6abb3ec4259a4881777212f637e53f1fce71L922 #449

File: src/modules/ChannelList/dux/__tests__/reducers.spec.js (hunk 3 above)
Comment (AhyoungRyu) [reply src/modules/ChannelList/dux/__tests__/reducers.spec.js]: Just changed the way of mocking functions https://github.com/sendbird/sendbird-uikit-react/pull/1124/commits/d691222b5b7c6e7f4cd78767cceb1a13579c972f#diff-9536f4716bc6385ea2e3e35374ac6abb3ec4259a4881777212f637e53f1fce71R912

File: src/utils/index.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (AhyoungRyu) [reply src/utils/index.ts]: 넵 https://github.com/sendbird/sendbird-uikit-react/pull/1124/commits/5f0b043c7e548a6e5f5e40474ead7a34d3901344 반영했습니다!

**Pull request merged**